sources/packages/backend/apps/api/src/testHelpers/testing-modules/testing-modules-helper.ts

```diff
@@ -23,7 +23,7 @@ export class CreateTestingModuleResult {
 export async function createTestingAppModule(): Promise<CreateTestingModuleResult> {
   await KeycloakConfig.load();
   const module: TestingModule = await Test.createTestingModule({
-    imports: [AppModule, createMockedZeebeModule(), MockedQueueModule],
+    imports: [AppModule, createZeebeModuleMock(), MockedQueueModule],
```

Comment:
Just checking. Any particular reason for changing the name to `createZeebeModuleMock` ?

Reply:
The class was moved to the lib and it seems more aligned with others.